src/write_gcm.c

```diff
@@ -2530,6 +2535,10 @@ static wg_context_t *wg_context_create(const wg_configbuilder_t *cb) {
     if (build->json_log_file == NULL) {
       WARNING("write_gcm: Can't open log file %s. errno is %d. Continuing.",
           cb->json_log_file, errno);
+      if (cb->log_partial_errors) {
```

Comment:
I'm following what the code does today. It seems to only log the response via wg_log_json_message in line 3859. What would you suggest to do different and where?